contents/blog/multivariated-testing-explained.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Multivariate testing, explained
+date: 2023-11-07
+author: ["liorn"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-company-culture-blog.png
+featuredImageType: full
+category: Engineering
+tags:
+  - Guides 
+  - Explainers
+  - Growth engineering
+  - Product engineers
+  - AB testing
+---
+
+## What is multivariate testing
+
+Multivariate testing is when you test multiple variables to determine which combination performs the best. 
+
+For example, let's say you want to test how both the color and placement of a button affect your conversion rate. You would then create four different variants to test:
+
+1. The control variant.
+2. The variant with only the color changed.
+3. The variant with only the placement changed.
+4. The variant with both the color and the placement changed.
+
+The results from the multivariate test would show not only which variant performs the best but also how each variable affected the outcome.
+
+It's similar to [A/B testing](/ab-testing), except in A/B testing you're only testing two different variants.
+
+![A/B testing vs multivariate testing](../images/blog/multivariated-testing-explained/ab-vs-mvt.png)
+
+## Benefits
+
+### 1. It saves you time
+
+Testing multiple combinations at the same time means you don't have to wait for many consecutive tests to finish. This would be the case if you were to run your tests using A/B tests, since you'd have to wait for each test before starting a new one.
+
+### 2. It uncovers interactions between variables
+
+Multivariate testing is not just about looking at which combination is the best; it's about understanding the weight each variable had in that success. This provides insights into understanding your holistic user experience.
+
+For example, let's say you're testing changes to the user dashboard of a B2B SaaS app. You decide to test how both the position of the navigation menu and the text of the buttons affect user engagement.
+
+In isolation, changing the button texts might affect how well users understand how to use your product. Similarly, moving the navigation menu from the top to the side could influence their ability to navigate efficiently.
+
+However, when you test these variables together, you may discover that their combined effect is much greater than each individual change.
```

Comment:
Hmm gave it some thought. I think since its an example, its okay to say "greater", and I think its clearer this way than saying "different' (or a something similar to different)
```suggestion
However, when you combine and test these changes together, you may discover that their combined effect is greater than each individual change.
```